docs/developers/symbol-information.md

```diff
@@ -107,6 +107,7 @@ Use `MethodSignature.parameterLists` to look up parameters of a method.
 def printMethodParameters(symbol: Symbol): Unit = {
   symbol.info.get.signature match {
     case signature @ MethodSignature(typeParameters, parameterLists, _) =>
+      println("signature  = " + signature)
```

Comment:
[changes in `ScalafixBuild`](https://github.com/scalacenter/scalafix/pull/1728/files#diff-cf70a614681abb9e33115a49e0b4a52865bdfcdb24c4c0df4af6546d92f88f5f) enabled more unused warnings on the builds